src/main/resources/validationrules/sscs/sscs-case-role-validation-legal-ops.drl

```diff
@@ -19,49 +19,94 @@ import java.time.ZonedDateTime;
 import java.time.temporal.ChronoUnit;
 
 /*
- * SSCS "tribunal-caseworker" case validation rule.
+ * SSCS "registrar" case validation rule.
  */
 
-rule "sscs_tribunal_caseworker_create_case_roles"
+rule "sscs_registrar_create_case_roles"
 when
     $f:  FeatureFlag(status && flagName == FeatureFlagEnum.SSCS_WA_1_0.getValue())
```

Comment:
Strange that it seems we already have FeatureFlagEnum.SSCS_WA_1_0 in the codebase before the changes for this branch. Any idea why?

Reply:
the SSCS config was initially done by the previous devs a year ago. back then, they used flags. maybe they can be retired?

Reply:
Matt is aware and happy with this reuse. resolved